features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2816,4 +2816,11 @@
     <AllowCrossTenantTokenIntrospection>{{oauth.allow_cross_tenant_token_introspection}}</AllowCrossTenantTokenIntrospection>
 
 
+    <!-- Claims which must not delete during existing claim mapping syncing process with idp claims. -->
+    <ClaimsNotToDelete>
```

Comment:
let's use "ImmutableClaimUris" for this config.
@IsuraD WDYT?

Reply:
Lets use IndelibleClaims.

Reply:
Explain the what IndelibleClaims in the comment section properly.